src/main/java/com/github/shatteredsuite/core/commands/predicates/HasItemPredicate.java

```diff
@@ -4,21 +4,22 @@
 import org.bukkit.entity.Player;
 import org.bukkit.inventory.ItemStack;
 
-import static java.util.Arrays.stream;
-
 public class HasItemPredicate extends SenderPlayerPredicate {
 
-    public HasItemPredicate(PredicateResponse response) {
+    private final int index;
+
+    public HasItemPredicate(PredicateResponse response, int index) {
         super(response);
+        this.index = index;
```

Comment:
What does having this index accomplish? Are we looking in a certain inventory slot? I'd take an `ItemStack` as the parameter and search for that in the inventory.

Reply:
Sorry, I misunderstood the requirement. The change has been pushed